tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3819,8 +3819,7 @@ public function getLastPublicSharePath():string {
 	 */
 	public function getPublicPreviewOfFile(string $fileName, string $token):void {
 		$url = $this->getBaseUrl() .
-			"/index.php/apps/files_sharing/ajax/publicpreview.php" .
-			"?file=$fileName&t=$token";
+			"/remote.php/dav/public-files/$token/$fileName?preview=1";
```

Comment:
This actually improves the code so that it uses a "real" documented API endpoint. And some test scenarios are passing - good.